setup.py

```diff
@@ -38,7 +38,12 @@ def find_version(*file_paths):
     scripts=[],
     packages=find_packages(exclude=['tests*']),
     package_data={
-        'botocore': ['cacert.pem', 'data/*.json', 'data/*/*.json'],
+        'botocore': [
+            'cacert.pem',
+            'data/*.json',
+            'data/*/*.json',
```

Comment:
I'm not exactly sure how this line has ever worked for including service model files because they're all one directory level down from what this indicates.

Reply:
Haven't tested this, but I suspect [this line in MANIFEST.in](https://github.com/boto/botocore/blob/develop/MANIFEST.in#L7) is responsible for the service model files. Both `package_data` and `MANIFEST.in` [are ok ways to define what files to include](https://setuptools.pypa.io/en/latest/userguide/datafiles.html#data-files-support), but using them both in parallel _does_ seem a bit confusing...

Reply:
I think Jonas is right, having the recursive include is more complete and keeping the relevant files partitioned to the MANIFEST.in file may be better. Can we get this change migrated over to follow the pattern set already for *.json?

Reply:
+1, let's move this to MANIFEST.in.

Technically, we could also use globs in `setup.py` with something like:

```
...
'data/**/*.json',
'data/**/*.json.gz`,
...
```

But @nateprewitt made me go look up when that feature was added to setuptools and it turns out to be less than half a year old: https://setuptools.pypa.io/en/latest/history.html#v62-3-0 (i.e. a bit too young to be ok to use for us)

Reply:
Ok thanks guys! I should be able to shoot up an update in the morning.